iceoryx_binding_c/include/iceoryx_binding_c/internal/cpp2c_publisher.hpp

```diff
@@ -30,6 +31,8 @@ struct cpp2c_Publisher
     cpp2c_Publisher& operator=(cpp2c_Publisher&& rhs) noexcept;
 
     iox::popo::PublisherPortData* m_portData{nullptr};
+    uint32_t m_customHeaderSize{iox::CHUNK_NO_CUSTOM_HEADER_SIZE};
+    uint32_t m_customHeaderAlignment{iox::CHUNK_NO_CUSTOM_HEADER_ALIGNMENT};
```

Comment:
No, this as just default values in case no custom header is used, i.e. 0 for size and 1 for alignment. There is always the restriction with alignment as power of two and size as multiple of alignment

Reply:
alignment 1 is unrestricted. Can be any multiple of 1. Considering it must be an integer value ... it can be any ;-)